src/reducers/gui.js

```diff
@@ -9,6 +9,7 @@ import targetReducer from './targets';
 import toolboxReducer from './toolbox';
 import vmReducer from './vm';
 import stageSizeReducer from './stage-size';
+import navigationReducer from './navigation';
```

Comment:
Wondering if this would ever be confused with overall navigation? i.e. usually the top bar on the website is the navigation bar (includes login etc). Perhaps it would be better named `tabsNavigationReducer`